cds_ils/ldap/errors.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019-2020 CERN.
+#
+# CDS-ILS is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+"""CDS-ILS ldap exceptions."""
+import sys
+
+
+class InvalidLdapUser(Exception):
+    """Invalid user exception."""
+
+    def __init__(self, *args, log_func=None):
+        """Constructor."""
+        super().__init__(*args)
+        print(args[0], file=sys.stderr)
```

Comment:
is `print` correct here? I saw it is used also above.

Reply:
it was used before, when the update/import is triggered from cli